src/command_local.cc

```diff
@@ -204,7 +204,7 @@ initialize_command_local() {
   CMD2_ANY         ("system.pid",      std::bind(&getpid));
 
   CMD2_VAR_C_STRING("system.api_version",           (int64_t)API_VERSION);
-  CMD2_VAR_C_STRING("system.client_version",        PACKAGE_VERSION);
+  CMD2_VAR_C_STRING("system.client_version",        VERSION);
```

Comment:
Why is this better?

Reply:
I much prefer PACKAGE_VERSION as it is more explicit.

Reply:
Done.